docs/snapcraft-parts.md

```diff
@@ -75,6 +75,36 @@ this in the snap directory, which is the final layout for the snap, this is
 where everything should look clean and crisp for a good quality snap.
 
 
+## Mastering the file system layout
+
+Snaps are typically deployed under `/snaps`, but because this location might
+change or multiple installation locations might be used (e.g.
+`/pre-installed` and `/snaps`), it's best not to hardcode it: to relate to
+resources shipped in your snap, use the `SNAP_DATA` environment variable.
```

Comment:
I'm not sure I understand this. What does `$SNAP_DATA` have to do with the /snaps directory?